multidim-interop/impl/kotlin/v0.2/Makefile

```diff
@@ -0,0 +1,20 @@
+image_name := kotlin-v0.1.0
+commitSha := 10eb5f4109a3fa789b38553079aa0736dfa58201
```

Comment:
Optional:

If you want to specify a branch instead of a commit sha, you can follow the pattern of creating a version.lock from here: https://github.com/libp2p/test-plans/pull/252/files#diff-ec9982cd1fe2ff166978a9f012d8c250aa7b2181d814eedd05dc2af191d02f89R20.

It creates a lock file that locks the expected sha256 of the contents of a branch.

Reply:
Thanks @MarcoPolo . Can you or @mxinden trigger a new build?
(https://github.com/libp2p/test-plans/actions/runs/5900727248)

Reply:
Thanks for triggering the build @MarcoPolo !
Almost... I see that rust-v0.52 --> kotlin-v.0.2 doesn't work yet
So, I have something to work on... :-)

Reply:
> If you put a '-' in front of a command in Makefile, it ignores when it fails. So, I do:

Today I learned. Thanks!